components/server/src/authorization/authorizer.ts

```diff
@@ -120,18 +117,60 @@ export class Authorizer {
         );
     }
 
+    async deleteOrganization(orgID: string): Promise<void> {
+        await this.authorizer.deleteRelationships(
+            v1.DeleteRelationshipsRequest.create({
+                relationshipFilter: v1.RelationshipFilter.create({
+                    resourceType: "organization",
+                    optionalResourceId: orgID,
+                }),
+            }),
+        );
+    }
+
+    async addOrganization(org: Organization, members: TeamMemberInfo[], projects: Project[]): Promise<void> {
+        const updates: v1.RelationshipUpdate[] = [];
+
+        for (let member of members) {
+            updates.concat(this.addOrganizationRoleUpdates(org.id, member.userId, member.role));
+        }
+
+        for (let project of projects) {
+            updates.concat(this.addProjectToOrgUpdates(org.id, project.id));
+        }
+
+        await this.authorizer.writeRelationships(
+            v1.WriteRelationshipsRequest.create({
+                updates: updates,
+            }),
+            {
+                orgID: org.id,
+            },
+        );
+    }
+
+    private addOrganizationRoleUpdates(orgID: string, userID: string, role: TeamMemberRole): v1.RelationshipUpdate[] {
+        if (role === "owner") {
+            return this.addOrganizationOwnerRoleUpdates(orgID, userID);
+        }
+        return [this.addOrganizationMemberRoleUpdates(orgID, userID)];
+    }
+
     private addOrganizationMemberRoleUpdates(orgID: string, userID: string): v1.RelationshipUpdate {
         return v1.RelationshipUpdate.create({
```

Comment:
should this be an array as well?

Reply:
For consistency? Or why?

It's not an array because it only requires a single update entry

Reply:
Ah, can change the name. I had it originally as all of them being arrays, but then removed the ones that didn't need to be.